hdmi2usb/modeswitch/boards.py

```diff
@@ -103,7 +104,10 @@ def firmware_path(filepath):
 class Board(BoardBase):
 
     def tty(self):
-        return self.dev.tty()
+        try:
+            return self.dev.tty()
+        except AttributeError:
+            return None
 
 
 def load_fx2(board, mode=None, filename=None, verbose=False):
```

Comment:
Can we move the libusb version of load_fx2 into libusb.py and load_fx2 which uses the binary into lsusb.py?